hops/featurestore.py

```diff
@@ -1753,7 +1753,8 @@ def setup_databricks(host, project_name, cert_folder="hops",
     
     write_init_script(dbfs_folder)
 
-    private_host = rest_rpc._get_hostname()
+    hive_endpoint = rest_rpc._get_featurestores()[0]['hiveEndpoint']
```

Comment:
Is this the correct dns name? Will it result in the internal or external one? I remember there being some troubles earlier.
I generally think during urlparse is a better idea.

Reply:
it's going to return the private IP of the instance. That should be ok, right? In any case you need to be able to route the to the private ip.

Reply:
I'm also working on https://github.com/logicalclocks/hopsworks-ee/pull/271 so that this code will be eventually removed.

Reply:
I tested urlparse, it doesn't work as here we just get `ip:port` and it sees it as part of the path, so you won't be able to get the hostname.
Honestly we'll get rid of this code. I think it's good enough with a split on `:`